scripts/package.js

```diff
@@ -0,0 +1,21 @@
+require('bluebird-global')
+const path = require('path')
+const fse = require('fs-extra')
+const { execute } = require('./utils/exec')
+
+const package = async () => {
+  const version = require(path.join(__dirname, '../package.json')).version.replace(/\./g, '_')
+
+  try {
+    await execute('yarn', { cwd: './packages/studio-be' })
+    await execute('cross-env pkg package.json', undefined, { silent: true })
```

Comment:
Just out of curiosity, is there any reason why cross-env is used here?

Reply:
Makes the command run on windows and on linux/mac without having to customize paths.
Eventually i'd like to simplify the packaging process even more, but each steps in its time